.github/workflows/tests.yml

```diff
@@ -71,7 +71,7 @@ jobs:
     runs-on: windows-2019
     strategy:
       matrix:
-        node: ['14', '12', '10']
+        node: ['10', '12', '14', '16']
```

Comment:
current versions are placed first since they run faster. that makes any errors reported during tests faster to report, which reduces wait time.

Reply:
it appears to be a common usage pattern I have seen to use an ascending order, incl. in the GH actions docs. that said, I didn't know that there was a purpose behind the order in this repo, but I looked at some sample data and that is simple not [always consistently] true - or better, if anything, it's randomly true, e.g.: https://github.com/rollup/rollup/runs/2211099843 since each GH action pod I assume does not always get the same exact resources assigned.

anyhow, if needed, I can change it back to descending order.

Reply:
I guess it does not matter then, thanks for checking!